content/quickstart-docker.ps1

```diff
@@ -0,0 +1,142 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Default values for SkyWalking versions
+$SW_VERSION = "9.7.0"
+$SW_BANYANDB_VERSION = "0.5.0"
+
+$env:BANYANDB_IMAGE = "apache/skywalking-banyandb:$SW_BANYANDB_VERSION"
+$env:OAP_IMAGE = "apache/skywalking-oap-server:$SW_VERSION"
+$env:UI_IMAGE = "apache/skywalking-ui:$SW_VERSION"
+
+# Unset the SW_STORAGE environment variable at the start of the script
+Remove-Item Env:\SW_STORAGE -ErrorAction Ignore
+
+# Ensuring script stops on error
+$ErrorActionPreference = "Stop"
+
+$storageOptionProvided = $false
+$detachOptionProvided = $false
+
+# Function to display usage information
+function Show-Usage {
+    Write-Host "Usage: quickstart-docker.ps1 [-h/--help] [-f] [--storage <storage_option>]"
+    Write-Host "Options:"
+    Write-Host "  -h/--help           About running the quickstart script without interaction"
+    Write-Host "  -d                  Run in background mode (docker compose up -d)"
+    Write-Host "  -f                  Run in foreground mode (docker compose up)"
```

Comment:
Can we only keep one of these options? For a Boolean option I don't think it necessary to provide 2 options

Reply:
Sure! Should've removed it, didn't wrap my mind around it.

Reply:
I can't see commits today. Do you miss it?

Reply:
> I can't see commits today. Do you miss it?

Just pushed, was having hotpot at home :)